README.md

```diff
@@ -148,6 +148,37 @@ code and results.
     (lambdaisland.deep-diff2.puget.color/document printer :keyword (str (.-unit value)))]))
 ```
 
+### Set up a custom print handler with different colors by utilizing Puget library
+
+Sometimes, we need to tune the colors because:
+
+- To ensure adequate contrast on a different background
+- To ensure readability by people who are colorblind
+- To match your editor or main diff tool's color scheme.
```

Comment:
```suggestion
Sometimes, we need to tune the colors to:

- Ensure adequate contrast on a different background.
- Ensure readability by people who are colorblind.
- Match your editor or main diff tool's color scheme.
```

Otherwise it read "we need to tune the colors because to ensure adequate...."